src/client/home/components/TrustedBySliver.tsx

```diff
@@ -1,13 +1,15 @@
 import React from 'react'
 import { Grid, Typography, createStyles, makeStyles } from '@material-ui/core'
-import trustedByMom from '@assets/components/home/trusted-by-sliver/1.png'
-import trustedByLta from '@assets/components/home/trusted-by-sliver/2.png'
-import trustedByMoh from '@assets/components/home/trusted-by-sliver/3.png'
-import trustedByMsf from '@assets/components/home/trusted-by-sliver/4.png'
-import trustedBySpf from '@assets/components/home/trusted-by-sliver/5.png'
-import trustedByIras from '@assets/components/home/trusted-by-sliver/6.png'
-import trustedByMoe from '@assets/components/home/trusted-by-sliver/7.png'
-import trustedByMha from '@assets/components/home/trusted-by-sliver/8.png'
+import trustedBy1 from '@assets/components/home/trusted-by-sliver/1.png'
+import trustedBy2 from '@assets/components/home/trusted-by-sliver/2.png'
+import trustedBy3 from '@assets/components/home/trusted-by-sliver/3.png'
+import trustedBy4 from '@assets/components/home/trusted-by-sliver/4.png'
+import trustedBy5 from '@assets/components/home/trusted-by-sliver/5.png'
+import trustedBy6 from '@assets/components/home/trusted-by-sliver/6.png'
+import trustedBy7 from '@assets/components/home/trusted-by-sliver/7.png'
+import trustedBy8 from '@assets/components/home/trusted-by-sliver/8.png'
+
+import i18next from 'i18next'
```

Comment:
could we shift this import so that it is together with the other library imports? thanks!

Reply:
Resolved here [13ac1f5](https://github.com/opengovsg/GoGovSG/pull/1846/commits/13ac1f5c07e39c0589a6a1c1b815f9720a94b6a9)